Header header

float32[] knots
float32[] coefficients
float32[] covariance
uint8 order
